source/Azure.Common/Variables/AzureVariableType.cs

```diff
@@ -0,0 +1,9 @@
+using Sashimi.Server.Contracts.Variables;
+
+namespace Sashimi.Azure.Common.Variables
+{
+    public static class AzureVariableType
+    {
+        public static readonly VariableType AzureAccount = new VariableType("AzureAccount");
```

Comment:
I didn't call this `VariableType` because it would be a pain to call it:
![image](https://user-images.githubusercontent.com/2888279/83367228-db9cf380-a3f6-11ea-8d4e-2fa7a83a4655.png)

Reply:
For backward compat I think we'll have to keep using "AzureAccount" as the literal string for this, but it's really being used as an AzureServicePrincipal by all things in Server these days, so could we make the VariableType and the ControlType clearly ServicePrincipal in the server world?